lib/active_record/connection_adapters/odbc_adapter.rb

```diff
@@ -50,15 +51,20 @@ class ODBCAdapter < AbstractAdapter
 
       def initialize(config_or_deprecated_connection, deprecated_logger = nil, deprecated_connection_options = nil, deprecated_config = nil, database_metadata = nil)
         super(config_or_deprecated_connection, deprecated_logger, deprecated_connection_options, deprecated_config)
-        @raw_connection = config_or_deprecated_connection
-
-        if database_metadata
-          @database_metadata = database_metadata
+        @config = deprecated_config
```

Comment:
It appears that in either case of the if conditional, we then reset `@config` to something. Do we still need this?

Reply:
No, good catch!